style.css

```diff
@@ -1,3 +1,220 @@
+body {
+  color: black;
+  background-color: #fefefe;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Crimson Text", serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 #root {
-  color: red;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 2rem;
```

Comment:
ooh css grid - I still haven't got round to learning that, you'll have to teach me some time

Reply:
Thanks for the useful feedback Dom